src/main/scala/swiss/dasch/api/ProjectsEndpoints.scala

```diff
@@ -131,6 +131,27 @@ final case class ProjectsEndpoints(base: BaseEndpoints) {
     )
     .tag(projects)
 
+  val postBulkIngestFinalize = base.secureEndpoint.post
+    .in(projects / shortcodePathVar / "bulk-ingest" / "finalize")
+    .out(jsonBody[ProjectResponse].example(ProjectResponse("0001")))
+    .description(
+      "Finalizes the bulk ingest. " +
+        "This will remove the files from the `tmp/<project_shortcode>` directory and the directory itself. " +
+        "This will remove also the mapping.csv file."
+    )
+    .tag(projects)
+
+  val getBulkIngestMappingCsv = base.secureEndpoint.get
+    .in(projects / shortcodePathVar / "bulk-ingest" / "mapping.csv")
+    .description(
+      "Get the current result of the bulk ingest, may be incomplete. " +
+        "The result is a csv with the following structure: " +
+        "original,derivative"
+    )
+    .out(stringBody)
+    .out(header(HeaderNames.ContentType, "text/csv"))
+    .tag(projects)
```

Comment:
I think it would be neat to also add a route to get the number of files in a `tmp/<shortcode>` directory to allow for easily finding out if an ingest was successful (I know it wouldn't work for cases where an asset cannot be ingested - but I don't think that's the normal case)

Reply:
Counting the files in a directory with potentially 100.000+ files is not cheap.